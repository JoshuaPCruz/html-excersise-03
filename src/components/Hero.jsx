import React from 'react';
import '../css/components/Hero.css';

const Hero = () => {
  return (
    <div className="Hero">
      <h1>Soluciones corporativas</h1>
      <h2>Lorem ipsum dolor sit amet, consectetuer adipiscing elit</h2>
    </div>
  );
};

export default Hero;