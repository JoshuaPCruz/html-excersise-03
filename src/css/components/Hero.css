.Hero {
  margin: 0 22px 45px;
  border-top: 2px solid var(--gray-primary);
  border-bottom: 2px solid var(--gray-primary);
  padding: 29px 0;
  text-align: center;
}

.Hero h1 {
  font-size: 72px;
  color: var(--gray-secondary);
}

.Hero h2 {
  font-size: 46px;
  color: var(--gray-primary);
}