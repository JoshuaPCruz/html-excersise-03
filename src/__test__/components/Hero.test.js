import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Hero from '../../components/Hero';

describe('<Hero />', () => {
  const hero = shallow(<Hero />);
  test('Render del componente Hero', () => {
    expect(hero.length).toEqual(1);
  });

  test('Render del wrapper de la hero', () => {
    expect(hero.find('h1').text()).toBe('Soluciones corporativas');
  });
});

describe('Hero Snapshot', () => {
  test('Comprobar la UI del componente Hero', () => {
    const hero = create(<Hero />);
    expect(hero.toJSON()).toMatchSnapshot();
  });
});