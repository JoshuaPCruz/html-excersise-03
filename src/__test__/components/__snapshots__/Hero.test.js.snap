// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Snapshot Comprobar la UI del componente Hero 1`] = `
<div
  className="Hero"
>
  <h1>
    Soluciones corporativas
  </h1>
  <h2>
    Lorem ipsum dolor sit amet, consectetuer adipiscing elit
  </h2>
</div>
`;
